example/android/build.gradle

```diff
@@ -6,14 +6,14 @@ buildscript {
         minSdkVersion = 16
         compileSdkVersion = 28
         targetSdkVersion = 28
-        supportLibVersion = "28.0.0"
+        supportLibVersion = "1.0.0"
```

Comment:
Why did you set this to version 1.0.0? Is this related to Android X?

Reply:
Yep. RN60 is AndroidX, so your project must go to AndroidX. The required steps in your project are to use Android Studio to [convert your build.gradle dependencies](https://github.com/react-native-community/react-native-share/pull/570/files#diff-15f694d688fe6360b46c5b8399a7eca8R174) and your local native code, and to change your [android.properties](https://github.com/react-native-community/react-native-share/pull/570/files#diff-b5cf0eed029ae59e02a6fb7ff8ef05b9R20) to turn on AndroidX and jetifier. 

Setting the version to 1.0.x here is what corresponds to the version numbers of the AndroidX libraries. 28.x versions are for the old support libraries.